### PR TITLE
Resetting the crawled url counter for AjaxSpider this solves #2610

### PR DESCRIPTION
**Pull request by Omkar20895**
- This PR addresses the issue [#2610](https://github.com/zaproxy/zaproxy/issues/2610) 
- Initially crawled url count has been set to zero.
- Every time a new scan starts the counter is reset to zero and then incremented accordingly.

### CODE REVIEW

**Comment by thc202** [comment]
- It seems that the problem is at [SpiderPanel.java#L400](https://github.com/zaproxy/zap-extensions/blob/3aaee82b25c9c986034d467dd5f72d48d198726f/src/org/zaproxy/zap/extension/spiderAjax/SpiderPanel.java#L400), instead of resetting the counter is incremented (also, it's missing the update of the label);
 - The issue reference in the commit message should have the zaproxy/zaproxy prefix, otherwise it's referencing an issue in zap-extensions repo.
 - Could you bump the version of the add-on (and update changes) in its [ZapAddOn.xml](https://github.com/zaproxy/zap-extensions/blob/77977cd13c6349cffa66330b10e42bcc1a17acb7/src/org/zaproxy/zap/extension/spiderAjax/ZapAddOn.xml) file?

**Comment by Omkar20895** [comment]
Yeah I wanted to point out this thing too as to why 'this.foundCount' is incremented on calling start scan. This adds an unnecessary increment to the url count. Instead of calling the function resetUrlCount that from extension Ajax.java I will replace the line you have pointed with the contents of resetUrlCount, this will fix the issue. Thanks

**Comment by Omkar20895** [comment]
And one more thing, I will update the version too and make the necessary changes.

**Comment by thc202** [comment]
Thank you. BTW, in the commit message you could say that it fixes the issue (not just referencing it). [1]

[1] https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by Omkar20895** [comment]
@thc202 shall I update the version in ZapAddOn.xml to '16' and also replace the data [here](https://github.com/zaproxy/zap-extensions/blob/77977cd13c6349cffa66330b10e42bcc1a17acb7/src/org/zaproxy/zap/extension/spiderAjax/ZapAddOn.xml#L10) with the issue that I have resolved?? Thanks.

**Comment by thc202** [comment]
Yes, version to 16 and remove (all) the old changes and add the new one.

**Comment by Omkar20895** [comment]
Sorry for the mess! I forgot to remove some commented code after completing the testing in the local repo. Thanks.

**Comment by kingthorin** [comment]
Could you to an interactive rebase and fixup into a single commit?

**Comment by thc202** [comment]
The space after the issue prefix needs to be removed (it should be all together).
In the change notes, it can have just the 1st note (the 2nd is a consequence of the 1st).

**Comment by Omkar20895** [comment]
I will squash all the commits above into 1. Thanks

**Comment by Omkar20895** [comment]
@thc202 @kingthorin please review this PR. Thanks

**Comment by kingthorin** [comment]
There seems to be some unrelated changes to the onlineMenu extension.

**Comment by thc202** [comment]
There are some entries in the commit message that should be removed (i.e. "[Update] ...", "[Final Update] ..." and "Latest files from Crowdin").

**Comment by Omkar20895** [comment]
- @kingthorin I had to pull some commits from the master before I had to commit my changes into the master, I think this might be the reason for that unrelated changes.
- @thc202 I have tried to squash them and resolved many merge conflicts and I guess I messed up my local repo.
- These conventions are new to me and I would keep that in mind from the next time. Can I please close this PR and send a new one? Sorry for the inconvenience, Thanks.

**Comment by kingthorin** [comment]
Ok, if you need to include more recent changes from Master in your Pull Request you should Rebase your feature branch based on Master.

Also don't worry about the Git learning curve, we've all experienced similar problems. I'll close this PR and you can raise a new one.

**Comment by thc202** [comment]
If you have any Git related question to address the pull request, just let us know.
